docker/docker-compose-logging.yml

```diff
@@ -0,0 +1,404 @@
+# Usage
```

Comment:
the logging file should only contain logflare and vector, don't need the other services in here.

docker-compose can load mutliple compose files using the `-f` flag, so we only need to include the 2 additional services in here.

Reply:
it will merge the two compose files together, so there is no need to include duplicate services.

Reply:
If so then we don't have to duplicate all config values, if the service names are the same they will get merged and override the previous log driver

Reply:
Will try it out